### PR TITLE
Deer

### PR DESCRIPTION
**Pull request by shayoki**
## About The Pull Request

This PR adds five new antler types made by me. And un-whitelists a clothing set by request.

## Why It's Good For The Game

More variety for deer and related characters with antlers will have more options to choose from, over the little selection there was. Additionally, the player 'Leafy' requested their donator items to no longer have restrictions. This also fulfills the long awaited request from the player 'ironknight' to fix the whitelist for their jacket.

Screenshots and local tests:

prev_short
Short Antlers

prev_complex
Complex Antlers

prev_large

Large Antlers

prev_regal
Regal Antlers

prev_monarch
Monarch Antlers

image
wow

## Changelog

:cl:
add: Five more new antler types.
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — APPROVED** [review]
Sprites are fine

**Comment by ReturnToZender** [comment]
It looks like you removed the items themselves rather than their whitelist

File: modular_zubbers/code/modules/customization/sprite_accessories/horns.dm [hunk 1]
```diff
@@ -52,3 +52,23 @@
...
+	name = "Regal Antlers"
+	icon_state = "antlersregal"
+
+/datum/sprite_accessory/horns/bubber/big/antlersmonarch
+	name = "Monarch Antlers"
+	icon_state = "antlersmonarch"
+
+/datum/sprite_accessory/horns/bubber/big/antlersless	//end of sprites by Gavla
+	name = "Short Antlers"
+	icon_state = "antlersless"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/customization/sprite_accessories/horns.dm:74]: Also please repath the icon states with snakecase

```suggestion
/datum/sprite_accessory/horns/bubber/big/antlers //sprites by Gavla
	name = "Aged Antlers"
	icon_state = "antlersmore"

/datum/sprite_accessory/horns/bubber/big/antlers/large
	name = "Large Antlers"
	icon_state = "antlersmost"

/datum/sprite_accessory/horns/bubber/big/antlers/regal
	name = "Regal Antlers"
	icon_state = "antlersregal"

/datum/sprite_accessory/horns/bubber/big/antlers/monarch
	name = "Monarch Antlers"
	icon_state = "antlersmonarch"

/datum/sprite_accessory/horns/bubber/big/antlers/short	//end of sprites by Gavla
	name = "Short Antlers"
	icon_state = "antlersless"
```

File: modular_zubbers/code/modules/customization/sprite_accessories/horns.dm [hunk 2]
```diff
@@ -52,3 +52,23 @@
 /datum/sprite_accessory/horns/bubber/halo //Sprites from Aurora Station
 	name = "Halo"
 	icon_state = "halo"
+
+/datum/sprite_accessory/horns/bubber/big/antlers //sprites by Gavla
+	name = "Aged Antlers"
+	icon_state = "antlersmore"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/customization/sprite_accessories/horns.dm:58]: Snakecase these, remember to do so in the DMI as well

```suggestion
	icon_state = "antlers_more"
```

File: modular_zubbers/code/modules/customization/sprite_accessories/horns.dm [hunk 3]
```diff
@@ -52,3 +52,23 @@
...
+
+/datum/sprite_accessory/horns/bubber/big/antlers/large
+	name = "Large Antlers"
+	icon_state = "antlersmost"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/customization/sprite_accessories/horns.dm:62]: Ditto

File: modular_zubbers/code/modules/customization/sprite_accessories/horns.dm [hunk 4]
```diff
@@ -52,3 +52,23 @@
...
+
+/datum/sprite_accessory/horns/bubber/big/antlers/regal
+	name = "Regal Antlers"
+	icon_state = "antlersregal"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/customization/sprite_accessories/horns.dm:66]: Ditto

File: modular_zubbers/code/modules/customization/sprite_accessories/horns.dm [hunk 5]
```diff
@@ -52,3 +52,23 @@
...
+
+/datum/sprite_accessory/horns/bubber/big/antlers/monarch
+	name = "Monarch Antlers"
+	icon_state = "antlersmonarch"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/customization/sprite_accessories/horns.dm:70]: Ditto

File: modular_zubbers/code/modules/customization/sprite_accessories/horns.dm [hunk 6]
```diff
@@ -52,3 +52,23 @@
...
+
+/datum/sprite_accessory/horns/bubber/big/antlers/short	//end of sprites by Gavla
+	name = "Short Antlers"
+	icon_state = "antlersless"
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/customization/sprite_accessories/horns.dm:74]: Ditto

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**